Dockerfile

```diff
@@ -81,4 +88,4 @@ VOLUME ["/data"]
 USER zenith
 EXPOSE 6400
 EXPOSE 9898
-CMD ["/bin/bash"]
+CMD ["/bin/bash"]
```

Comment:
> no newline in the end

It's not like I'm advocating to that, but the majority of our files end with a newline, for example, `Cargo.toml` above and that was not changed in the PR, so let's keep things aligned and newlined here.